docker-compose.yaml

```diff
@@ -0,0 +1,12 @@
+version: "3.8"
+
+services:
+  pb-go:
+    build:
+      context: .
+    entrypoint:
+    - /go/bin/go
+    volumes:
+    - type: bind
+      source: ./examples/sqlitetst
```

Comment:
I think it might be better to mount both examples and show how to build one or the other from the same container.

Reply:
Okay. Consider it done.